src/components/outline-nav/components/outline-list-items/index.tsx

```diff
@@ -0,0 +1,35 @@
+import { OutlineLink, OutlineLinkWithNesting } from '..'
+import type { OutlineLinkItem } from 'components/outline-nav/types'
+
+/**
+ * Render an array of OutlineNav list items.
+ *
+ * Each item can be either a plain link item, or
+ * a link item with nested links.
+ *
+ * All items are wrapped in an <li /> element.
+ */
+function OutlineListItems({ items }: { items: OutlineLinkItem[] }) {
+	return (
+		<>
+			{items.map((item: OutlineLinkItem) => {
+				const uniqueKey = `${item.title}${item.url}`
```

Comment:
🙋🏻 Do we need both `title` and `url` in the key or is `url` enough? There shouldn't be any heading elements on a page with duplicate `id`s. If we do have cases of this, it'd be good to have an Asana ticket for fixing those, and a comment here as a breadcrumb!

Reply:
Ah true, `url` should be enough! I'll update accordingly 👍